extension/landing-zone/README.md

```diff
@@ -0,0 +1,23 @@
+# VSI add-on for landing-zone
+
+A module creating a VSI in an existing landing zone VPC.
```

Comment:
I wonder if module is the correct word here. It is more of a solution. Technically it is a deployable architectre extension - @SirSpidey any suggestions?